worker/src/DepOpenSSL.cpp

```diff
@@ -22,6 +23,14 @@ void DepOpenSSL::ClassInit()
 	  []
 	  {
 		  MS_DEBUG_TAG(info, "openssl version: \"%s\"", OpenSSL_version(OPENSSL_VERSION));
+		  MS_DEBUG_TAG(info, "openssl CPU info before init: \"%s\"", OpenSSL_version(OPENSSL_CPU_INFO));
+
+		  // Initialize.
+		  // It calls OPENSSL_init_crypto() which calls OPENSSL_cpuid_setup()
+		  // filling the capability bits used for hardware encryption.
+		  OPENSSL_init_ssl(0 /*opts*/, nullptr /*settings*/);
+
+		  MS_DEBUG_TAG(info, "openssl CPU info after init : \"%s\"", OpenSSL_version(OPENSSL_CPU_INFO));
```

Comment:
Wrong space before colon.

Reply:
Fixed, `OPENSSL_init_ssl` and this log have been removed.